exercises/practice/difference-of-squares/.meta/proof.ci.wren

```diff
@@ -0,0 +1,39 @@
+class Squares {
+  static squareOfSum(n) {
+    var sum = 0
+    var x = 0
+
+    while (x < n) {
+      x = x + 1
+      sum = sum + x
+    }
+    
+    return sum * sum
+  }
+
+  static sumOfSquares(n) {
+    var squares = 0
+    var x = 0
+
+    while (x < n) {
+      x = x + 1
+      squares = squares + (x * x)
+    }
+
+    return squares
+  }
+
+  static differenceOfSquares(n) {
```

Comment:
Why doesn't this just delegate to the other two functions?

Reply:
I've noticed in other exercises that the proof code isn't generally the best way to do it, so I kind of just went with it.

Reply:
_Lets replace it with that, please._  Note: That ALSO isn't optimal - since you can do this in constant time with simple math... but using loops is ok for the proof... (and more likely to be what students might do at this stage) but the subtraction IMHO should really reuse the existing functions or it's repeating for no reason at all.

Reply:
Okay, I've replaced it with that, now. Unnecessary code duplication doesn't teach good habits.